From 0780181a3285511f166487a54ebc231fc657edfe Mon Sep 17 00:00:00 2001
From: Catalin Enache <catalin.enache@windriver.com>
Date: Mon, 25 Jul 2016 16:38:51 +0300
Subject: [PATCH] Install missing conf file

openvzutilstest.conf file is needed by openvzutilstest test.

Upstream-Status: Inapproriate

Signed-off-by: Catalin Enache <catalin.enache@windriver.com>
[KK: Update context for 1.3.5.]
Signed-off-by: Kai Kang <kai.kang@windriver.com>
[MA: Update context for v4.3.0]
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
---
 tests/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 65f776e..8a6fd44 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -173,6 +173,7 @@ EXTRA_DIST = \
 	$(NULL)
 
 test_helpers = commandhelper ssh
+test_misc =
 test_programs = virshtest sockettest \
 	virhostcputest virbuftest \
 	commandtest seclabeltest \
@@ -305,6 +306,7 @@ endif WITH_LXC
 
 if WITH_OPENVZ
 test_programs += openvzutilstest
+test_misc += openvzutilstest.conf
 endif WITH_OPENVZ
 
 if WITH_ESX
@@ -1488,7 +1490,7 @@ endif ! WITH_CIL
 
 buildtest-TESTS: $(TESTS) $(test_libraries) $(test_helpers)
 
-PTESTS = $(TESTS) $(test_helpers) test-lib.sh virschematest
+PTESTS = $(TESTS) $(test_helpers) $(test_misc) test-lib.sh virschematest
 
 install-ptest:
 	list='$(TESTS) $(test_helpers) test-lib.sh virschematest'

-- 
2.7.4

